libraries/helpers.rb

```diff
@@ -30,15 +30,23 @@ def default_user_attributes
   end
 
   # Helpers method for recursor feature
-  module PdnsRecursorHelpers
+  module RecursorHelpers
     include Pdns::Helpers
 
-    def systemd_name(name = nil)
-      "pdns-recursor@#{name}"
+    def systemd_name(name = '')
+      if name.empty?
+        'pdns-recursor'
+      else
+        "pdns-recursor@#{name}"
+      end
     end
 
-    def sysvinit_name(name = nil)
-      "pdns_recursor-#{name}"
+    def sysvinit_name(name = '')
+      if name.empty?
+        'pdns-recursor'
+      else
+        "pdns-recursor-#{name}"
```

Comment:
According to the documents this shouldn't have the second hyphen and should probably be `"pdns-recursor_#{name}` correct?